crypto/evp/p_lib.c

```diff
@@ -878,6 +916,14 @@ static void evp_pkey_free_it(EVP_PKEY *x)
 #endif
 }
 
+static void evp_pkey_free_it(EVP_PKEY *x)
+{
+    /* internal function; x is never NULL */
```

Comment:
Sounds like a dangerous assumption

Reply:
That has always been the assumption (scroll up a little and you'll see that this comment is nothing new)
This is an *internal **static*** function...  if the caller doesn't get it right, the fault is none other than our own, and I expect there to be a [KABOOM] in our tests if we mes up.